aml-development/frontend
# feat(listing mgmt): table view

AMLOS-688

**Description**
Replicate the Listing Management grid view in table form.

**Acceptance Criteria**
- The user can switch between Grid View and Table View on the Listing Management page.
- The user should be able to Edit/View/Delete a listing from table view.
- See AML 2.0 Table View for ideas.

**How to Test**
- Pull this branch
- Go to listing management as different users (bigbrother, bettafish, etc)
- Verify that you can toggle between grid/table view on different sections (my listings, all listings, org listings)
- Verify that table sorts
- Verify that table actions (edit, view, delete, toggle featured, etc) are performed as expected

AMLOS-745

**Description**
As a user, I would like the ability to load items in small chunks, loading more as necessary.

**Acceptance Criteria**
- User can load next and prev from each view in listing management
- Loading next and prev updates the current page the user is viewing
- Only applies to table view

**How to Test**
- Pull this branch
- Go to listing management as different users (bigbrother, bettafish, etc)
- Toggle view to open up the table view.  Verify that the table always has 24 rows.
- If there is a next set of listings to be loaded, verify that the _Next_ button is present and that clicking it loads the next page.
- If there is a previous set of listings to be loaded, verify that the _Previous_ button is present and that clicking it loads the previous page.
